chef/cookbooks/nova/templates/default/libvirtd.conf.erb

```diff
@@ -246,38 +255,48 @@ auth_tcp = "<%= @libvirtd_auth_tcp %>"
 
 # The maximum number of concurrent client connections to allow
 # over all sockets combined.
-#max_clients = 20
+max_clients = 5000
 
+# The maximum length of queue of connections waiting to be
+# accepted by the daemon. Note, that some protocols supporting
+# retransmission may obey this so that a later reattempt at
+# connection succeeds.
+#max_queued_clients = 1000
+
+# The maximum length of queue of accepted but not yet not
+# authenticated clients. The default value is zero, meaning
+# the feature is disabled.
+#max_anonymous_clients = 20
 
 # The minimum limit sets the number of workers to start up
 # initially. If the number of active clients exceeds this,
 # then more threads are spawned, up to max_workers limit.
 # Typically you'd want max_workers to equal maximum number
 # of clients allowed
-#min_workers = 5
-#max_workers = 20
+min_workers = <%= @libvirtd_min_workers %>
+max_workers = <%= @libvirtd_max_workers %>
 
 
 # The number of priority workers. If all workers from above
-# pool will stuck, some calls marked as high priority
+# pool are stuck, some calls marked as high priority
 # (notably domainDestroy) can be executed in this pool.
-#prio_workers = 5
+prio_workers = 5
 
 # Total global limit on concurrent RPC calls. Should be
 # at least as large as max_workers. Beyond this, RPC requests
-# will be read into memory and queued. This directly impact
+# will be read into memory and queued. This directly impacts
 # memory usage, currently each request requires 256 KB of
 # memory. So by default up to 5 MB of memory is used
 #
 # XXX this isn't actually enforced yet, only the per-client
 # limit is used so far
-#max_requests = 20
+max_requests = <%= @libvirtd_max_workers %>
 
 # Limit on concurrent requests from a single client
 # connection. To avoid one client monopolizing the server
 # this should be a small fraction of the global max_requests
 # and max_workers parameter
-#max_client_requests = 5
+max_client_requests = <%= @libvirtd_min_workers %>
```

Comment:
@vuntz would it make you more happy to use a different variable name iwth the same value? that is probably cleaner.